sdv/constraints/tabular.py

```diff
@@ -301,22 +312,23 @@ def _fit(self, table_data):
                 The Table data.
         """
         if self._high_is_scalar is None:
-            self._high_is_scalar = self._high not in table_data.columns
+            self._high_is_scalar = not isinstance(self._high, list)
         if self._low_is_scalar is None:
-            self._low_is_scalar = self._low not in table_data.columns
+            self._low_is_scalar = not isinstance(self._low, list)
 
         if self._high_is_scalar and self._low_is_scalar:
             raise TypeError('`low` and `high` cannot be both scalars at the same time')
         elif self._low_is_scalar:
-            self.constraint_columns = (self._high,)
-            self._dtype = table_data[self._high].dtype
+            self.constraint_columns = tuple(self._high)
         elif self._high_is_scalar:
-            self.constraint_columns = (self._low,)
-            self._dtype = table_data[self._low].dtype
+            self.constraint_columns = tuple(self._low)
         else:
-            self._dtype = table_data[self._high].dtype
+            self.constraint_columns = tuple(self._low + self._high)
+            if len(self.constraint_columns) > 2:
+                raise ValueError('`low` and `high` cannot be more than one column.')
 
         self._column_to_reconstruct = self._get_column_to_reconstruct()
+        self._dtype = [table_data[column].dtype for column in self._column_to_reconstruct]
```

Comment:
Should we rename to `self._columns_to_reconstruct` and `self._get_columns_to_reconstruct`?

Reply:
Sounds good, I believe that the same change should also apply to `diff_column` to `diff_columns`.